### ISSUE TITLE
[Markup and meta data] HTML markup isn't valid. (09.01.8)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#29348]
#### VFS team name
Education Application (BAH)

#### VFS product name
Comparison Tool Redesign

#### Point of Contact/Reviewers
Trevor Pierce (Accessibility)

---

### Platform Issue
HTML markup isn't valid.

### Issue Details
Compare Institutions label doesn't announce school added to / removed from the comparison tray to screen readers.

### Link, screenshot or steps to recreate
![Screen Shot 2021-08-26 at 5 14 25 PM](https://user-images.githubusercontent.com/934879/131043174-0211e52a-39f3-4675-aaa7-c63331b26166.png)

### VA.gov Experience Standard
[Cateogy Number 09, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A

---

### Platform Recommendation
Add a live region to the heading / button used to toggle the Compare Institutions. Also consider an SR-only snippet that can be updated if you are adding a school or removing one. Ideally the screen reader would announce "Florida State added. Compare Instutions, three of three." or "Florida State removed. Compare Institutions, two of three."

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by dmccurdybah** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/18439

**Comment by mattpointzxer0** [comment]
@1Copenut and @noahgelman, this appears to be done! Verify and close when you're able? Thank you!

**Comment by dneel-bah** [comment]
Per agreement with OIT/OCTO PMs this issue passed internal testing with assistive technology.  Closing this issue as completed

**Comment by noahgelman** [comment]
LGTM
